Add Spanish Language Support

clostao opened the pull request.

nazar-pc commented on res/translations/es/messages.ftl:
```diff
@@ -0,0 +1,211 @@
+loading_configuration_title = Cargando configuración
+loading_configuration_step_loading = Cargando configuración...
+loading_configuration_step_reading = Leyendo configuración...
+loading_configuration_step_configuration_exists = Leyendo configuración...
+loading_configuration_step_configuration_not_found = Leyendo configuración...
```
These should change despite them being the same string in English. See the key for hints on what it should be.

nazar-pc commented on res/translations/es/messages.ftl:
```diff
@@ -0,0 +1,211 @@
+    ✔ Una cartera donde recibirás las recompensas (utiliza Subwallet, la extensión de polkadot{”{”}.js{”}”} o cualquier otra cartera compatible con la cadena Substrate)
+    ✔ 100 GB de espacio en un SSD de buena calidad para almacenar los datos del nodo
+    ✔ Cualquier SSD (o varios) con tanto espacio como puedas permitirte para fines de agricultura, esto es lo que generará las recompensas
+
```
Should be no new line hehe

nazar-pc reviewed (commented):
Thanks for adding this. Please check formatting, we ensure everything is valid at compile time here.

nazar-pc commented:
Also see changes in last commit of https://github.com/autonomys/space-acres/pull/288

clostao commented:
> @clostao PR was open for about two weeks, do you plan to finish it?

My bad, I forgot about it

nazar-pc commented:
Can you update the file locally and push a single commit instead of multiple "Update messages.ftl", please?

Each commit triggers useless email notification for me.

nazar-pc commented:
I squashed all the commits, but it still doesn't compile, meaning there are issues in translation file and it doesn't match other translations (this is why we check it at compile time).

clostao commented:
> I squashed all the commits, but it still doesn't compile, meaning there are issues in translation file and it doesn't match other translations (this is why we check it at compile time).

It is now solved, I've also added the two other new entries and now compiles fine.

nazar-pc reviewed (approved):
Awesome, thanks!
